languages/csharp/exercises/concept/exceptions/.meta/Example.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+static class SimpleCalculator
+{
+    public static string ErrorLog {get; set;}
+    public static int Calculate(int operand1, int operand2, string operation)
+    {
+        int result = 0;
+        ErrorLog = "";
```

Comment:
What if instead of returning an integer and having the error log as a property, we would just return a string that describes the result? Something like this:

```csharp
SimpleCalculator.Calculate(1, 2, "+")
// => "1 + 2 = 3"
```

This would make things a little bit more elegant and allow us to remove the `ErrorLog` property.

Reply:
This is a really good idea. In retrospect I'm not sure why I didn't go down that route in the first place.

Reply:
No worries! This is precisely why PR reviews can be very helpful. We constantly do this for all new exercises :)